Description: Build libkms against in-tree xf86drm.h header
 libkms requires xf86drm.h, and uses symbols from libdrm.  Add
 an appropriate search path to CFLAGS, and libdrm to LIBADD.
Author: Christopher Halse Rogers <christopher.halse.rogers@canonical.com>
Bug: https://bugs.freedesktop.org/show_bug.cgi?id=26852

Index: libdrm/libkms/Makefile.am
===================================================================
--- libdrm.orig/libkms/Makefile.am	2010-05-26 16:35:15.092706440 +1000
+++ libdrm/libkms/Makefile.am	2010-05-26 16:42:41.722739824 +1000
@@ -1,11 +1,12 @@
 AM_CFLAGS = \
 	$(WARN_CFLAGS) \
-	-I$(top_srcdir)/include/drm
+	-I$(top_srcdir)/include/drm \
+	-I$(top_srcdir)
 
 libkms_la_LTLIBRARIES = libkms.la
 libkms_ladir = $(libdir)
 libkms_la_LDFLAGS = -version-number 1:0:0 -no-undefined
-libkms_la_LIBADD =
+libkms_la_LIBADD = ../libdrm.la
 
 #if HAVE_LIBUDEV
 #libkms_la_LIBADD += $(LIBUDEV_LIBS)
